weblate_web/templates/user.html

```diff
@@ -14,47 +17,56 @@ <h1 class="section-title min-m">{% trans "User profile" %}</h1>
             </div>
             {% with services=user.service_set.all %}
             {% if services %}
-            <h1>{% trans "Your services" %}</h1>
-            {% for service in services %}
+            <div class="updated-service-title">
+                <h1>{% trans "My services" %}<span><a href="https://hosted.weblate.org/accounts/profile/">{% trans "See user profile " %}<img src="{% static "img/ArrowR.png" %}" alt=">" /></a></span></h1>
+            </div>
+                {% for service in services %}
```

Comment:
Unnecessary indentation change:

```suggestion
            {% for service in services %}
```

Reply:
Yes, it was unnecessary. I have gotten rid of it now.